public/components/freelancer.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+/**
+ * Freelancer View Components
+ * CSS styling in css/freelancer.css
+ */
+
+ajaxRequest("GET", window.location, { ajax : true }, {}, renderComponent);
+
+function renderComponent(data) {
+  // freelancer info
+  const tags = data.tags;
+  const listTags = tags.map((tag, index) =>
+    <li key={index}>
+      {tag.tagName}
+    </li>
+  );
+
+  ReactDOM.render(
+    <FreelancerView
+      first={data.firstName}
```

Comment:
For all the following lines, does spacing around `=` work or is there none because it's HTML?